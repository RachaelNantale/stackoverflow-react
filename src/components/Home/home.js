import React from 'react';

const Home = () => (
  <div className="container">
    <p>Hello World</p>
  </div>
);


export default Home;
