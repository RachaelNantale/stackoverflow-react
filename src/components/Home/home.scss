.header-img {
  width: 100%;
  height: 20rem;
  overflow: hidden;
}
img {
  width: 100%;
}

@media all and (max-width: 800px) {
  .header-img {
    height: 10rem;
  }
  .signup-container {
    background: white !important;
    
    .centered-form {
    box-shadow: none !important;
    border-radius: 5px;
    margin: 100px 10px 10px 10px !important;
    border: 1px solid #dddddd;
    color: #000 !important;
    }
  } 
}