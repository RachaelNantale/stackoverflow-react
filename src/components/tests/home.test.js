import React from 'react';
import { shallow } from 'enzyme';
import Home from '../Home/home';

it('renders the signup component', () => {
  const HomeComponent = shallow(<Home />);
  expect(HomeComponent).toMatchSnapshot();
});
