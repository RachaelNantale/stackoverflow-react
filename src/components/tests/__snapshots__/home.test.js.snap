// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the signup component 1`] = `
<div
  className="container"
>
  <p>
    Hello World
  </p>
</div>
`;
